### PR TITLE
rh-che #1116: Adding env var which allows to disable CORS

### PR DESCRIPTION
**Pull request by ibuziuk**
The plan is the following:

- disable CORS on prod-preview, but not on prod
- test if everything works fine on prod-preview
- disable CORS  on prod

### What does this PR do?
Adding env var which allows to disable CORS

### What issues does this PR fix or reference?
https://github.com/redhat-developer/rh-che/issues/1116

### HK issue
https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/2470

### CODE REVIEW

**Comment by ibuziuk** [comment]
Blocked until cm would be updated as part of the https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/2470

File: openshift/rh-che.app.yaml [hunk 1]
```diff
@@ -85,6 +85,16 @@ objects:
               configMapKeyRef:
                 key: che-jsonrpc-max-pool-size
                 name: rhche
+          - name: CHE_CORS_ENABLED
+            valueFrom:
+              configMapKeyRef:
+                key: che-cors-enabled
+                name: rhche
+          - name: CHE_WSAGENT_CORS_ALLOWED__ORIGINS
```
Comment (amisevsk) [inline openshift/rh-che.app.yaml]: (Totally optional) suggestion: should we move this env entry down to keep the env var names alphabetically ordered? It doesn't matter too much for just one yaml but I've found it useful when comparing against the minishift addon template.

Comment (ibuziuk) [reply openshift/rh-che.app.yaml]: no IMO, since those env var would be removed in the `6.16.0` release - CORS is planned to be always disabled

Comment (ibuziuk) [reply openshift/rh-che.app.yaml]: ah, sorry last comment was for `Should we also add these values to the minishift template yaml? (I don't know if it's interesting/useful to do testing there)`. Sure thing I will move the env var down ;-)

Comment (ibuziuk) [reply openshift/rh-che.app.yaml]: @amisevsk should be better now

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
[test]

**Pull request merged**